{"name": "test_user_login_failure[chromium]", "status": "passed", "description": "\n    Test case to verify failed login using invalid credentials.\n    ", "parameters": [{"name": "browser_type", "value": "'chromium'"}], "start": 1727301892699, "stop": 1727301893648, "uuid": "987f877b-9780-4da9-9e2e-a019899e2efa", "historyId": "df7edcb30dcc084392721104227917d7", "testCaseId": "17d3089194985caa2bc4fdc96dc1b569", "fullName": "tests.test_user_auth#test_user_login_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_auth"}, {"name": "host", "value": "MarvelDeviser"}, {"name": "thread", "value": "14264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_auth"}]}